packages/harpguru-core/src/components/option/option-styles.ts

```diff
@@ -1,30 +1,59 @@
 import { State } from 'react-native-gesture-handler'
 import { StyleSheet, ViewStyle, TextStyle } from 'react-native'
 
-import { getSizes } from '../../styles'
+import { getSizes, colors } from '../../styles'
 
 type OptionStyles = {
   readonly option: ViewStyle
   readonly optionTitle: TextStyle
-  readonly optionValue: TextStyle
+  readonly optionValues: ViewStyle
+  readonly distantOptionValue: TextStyle
+  readonly nextOptionValue: TextStyle
+  readonly activeOptionValue: TextStyle
 }
 
 export const getStyles = (): OptionStyles => {
   const sizes = getSizes()
-  const { 7: variableSize, 8: titleSize } = sizes
 
   const styles = StyleSheet.create({
     option: {
       flex: 1,
       flexDirection: 'column',
       alignItems: 'center',
-      justifyContent: 'space-evenly',
+      justifyContent: 'space-between',
     },
     optionTitle: {
-      fontSize: titleSize,
+      alignSelf: 'center',
+      fontSize: sizes['8'],
+      paddingTop: sizes['10'],
+      textDecorationLine: 'underline',
     },
-    optionValue: {
-      fontSize: variableSize,
+    optionValues: {
+      paddingBottom: sizes['9'],
+    },
+    distantOptionValue: {
```

Comment:
There's a lot of repetition in these option value related styles. There could maybe be a generic `optionValue` styles which they are all combined with in the `styles={[style, style]}` format in the jsx.

Reply:
Raised as tech debt in #20